src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RENTAL_RATE = 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DAYS = 7;
+  const SHORT_TERM_DAYS = 3;
+
```

Comment:
You can create a variable for base price and use it everywhere
```suggestion
  const SHORT_TERM_DAYS = 3;
  const BASE_PRICE = days * DAILY_RENTAL_RATE;
```